test/unit/assorted/server_discovery_and_monitoring.spec.test.js

```diff
@@ -56,11 +56,9 @@ describe('Server Discovery and Monitoring (spec)', function () {
     serverConnect.restore();
   });
 
-  // DRIVERS-1249 should add directConnection and then update spec, remove skip
   const shouldSkip = desc => {
     const descriptions = [
```

Comment:
This all LGTM for srv polling, we could merge just that and come back to SDAM in another PR.